### PR TITLE
Updatefont

### PR DESCRIPTION
**Pull request by ivesfelix**
Updated font styles & made paragraphs narrower.

### CODE REVIEW

File: client/assets/style/style.css [hunk 1]
```diff
@@ -1,20 +1,49 @@
...
+  color: #f3785b;
+  text-decoration: none;
+}
+
+a:hover {
+  color: #f8b4a4;
+  text-decoration: none;
+}
+
+.h2, h2 {
```
Comment (JoeDou) [inline client/assets/style/style.css]: i don't think we have any .h2 classes so h2 should just be good enough, same for h3 and h4

File: client/assets/style/style.css [hunk 2]
```diff
@@ -58,44 +86,51 @@ html, body {
 .text-view {
   min-height: 50vh;
   background-color: white;
-  margin: 20px;
-  padding: 15px;
-  border: 2px solid black;
-  border-radius: 25px;
+  margin: 20px 300px 20px 300px;
```
Comment (JoeDou) [inline client/assets/style/style.css]: if top & bottom, right & left are the same, you should consolidate the two.  margin: 20px 300px.  This goes for padding as well
http://www.w3schools.com/css/css_margin.asp for more info

**Comment by JoeDou** [comment]
made a few comments, if you can update those, I'll merge the PR

File: client/assets/style/style.css (hunk 1 above)
Comment (ivesfelix) [inline client/assets/style/style.css]: resolved

File: client/assets/style/style.css (hunk 2 above)
Comment (ivesfelix) [inline client/assets/style/style.css]: resolved. thanks!

**Pull request merged**